ui/src/main/webapp/src/app/app.routing.ts

```diff
@@ -71,16 +71,15 @@ export const appRoutes: Routes = [
                 component: DefaultLayoutComponent,
                 children: [
                     {path: '', redirectTo: "/project-list", pathMatch: "full"},
+                    {path: 'about', component: AboutPageComponent},
                     {
                         path: "configuration",
-                        resolve: { configuration: ConfigurationResolve },
```

Comment:
Sure, I will try. I admit that moving one line makes the PR completely unreadable.